share/instruments/instruments.xml

```diff
@@ -7933,7 +7933,7 @@
                         <head>cross</head>
                         <line>0</line>
                         <voice>0</voice>
-                        <name>Chinese Cymbal</name>
+                        <name>Tam-tam</name>
```

Comment:
A few lines above this we have:

```xml
                  <trackName>Tam-Tam</trackName>
                  <longName>Tam-Tam</longName>
                  <shortName>Tam</shortName>
```

Notice "Tam-Tam" (capital 'T') vs. "Tam-tam" (lowercase 't'). Maybe we should be consistent about this?

On the other hand, we already have "Chinese Tom-Toms" in the score and "Chinese Tom-toms" in the drumset palette.

Reply:
Ignoring for now since drum names don't appear in the score anyway.

Reply:
Translatability issue, see above